dnSpy/dnSpy.Contracts.DnSpy/Controls/GridViewSortableColumn.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+
+using dnSpy.Contracts.MVVM;
+
+namespace dnSpy.Contracts.Controls {
+	/// <summary>
+	/// GridViewColumn with sort behaviour
+	/// </summary>
+	public class GridViewSortableColumn : GridViewColumn {
+		#region Dependency properties
+
+		/// <summary>
+		/// Sort command to be specified for parent ListView
+		/// </summary>
+		public static readonly DependencyProperty SortCommandProperty;
+		/// <summary>
+		/// Item model property displayed in this column
+		/// </summary>
+		public static readonly DependencyProperty SortByProperty;
+		/// <summary>
+		/// Sort direction for this column
+		/// </summary>
+		public static readonly DependencyProperty SortDirectionProperty;
+
+		static GridViewSortableColumn() {
+			// Attached to ListView
+			SortCommandProperty = DependencyProperty.RegisterAttached(
+				"SortCommand",
+				typeof(ICommand),
+				typeof(GridViewSortableColumn),
+				new PropertyMetadata(SortCommandPropertyChanged));
+			SortByProperty = DependencyProperty.Register(
+				nameof(SortBy),
+				typeof(string),
+				typeof(GridViewSortableColumn),
+				null);
+			SortDirectionProperty = DependencyProperty.Register(
+				nameof(SortDirection),
+				typeof(SortDirection?),
+				typeof(GridViewSortableColumn),
+				new FrameworkPropertyMetadata(null, FrameworkPropertyMetadataOptions.AffectsRender | FrameworkPropertyMetadataOptions.AffectsMeasure, null));
+		}
+
+		/// <summary>
+		/// Getter for SortCommand attachable property
+		/// </summary>
+		public static ICommand GetSortCommand(DependencyObject element) {
+			return (ICommand)element.GetValue(SortCommandProperty);
```

Comment:
C#7 can be used now, so `=>` is preferable if it's just a oneline method or getter/setter.